src/Heap.sol

```diff
@@ -160,26 +160,26 @@ library BasicHeap {
             // Find the child with largest value.
             unchecked {
```

Comment:
(optional) we could move this `unchecked` outside of the `while` loop. It should not change anything but maybe easier to read because it would not mix with the logic of the loop (and also we have done that for the other functions such as `_shiftUp`)